### PR TITLE
Issuue 1

### PR DESCRIPTION
**Pull request by anuragdw710**

### CODE REVIEW

**Comment by bwanglzu** [comment]
can you give a descriptive title and PR description?

File: daemon/clients/base.py [hunk 1]
```diff
@@ -35,48 +33,44 @@ def __init__(
         timeout: int = None,
     ):
         self._logger = logger
-        self.timeout = aiohttp.ClientTimeout(
-            timeout if isinstance(timeout, Number) else 10 * 60
-        )
+        self.timeout = timeout
```
Comment (jacobowitz) [inline daemon/clients/base.py:36]: You are effectively removing the timeout usage in your PR, why is that?

File: daemon/clients/base.py [hunk 2]
```diff
@@ -116,10 +107,24 @@ async def list(self) -> Dict:
                     f'got response {response.status} while listing all {self._kind}s'
                 )
 
+    @if_alive
+    async def clear(self):
+        """Send delete request to api
```
Comment (jacobowitz) [inline daemon/clients/base.py:112]: Please be a bit more specific here

**Review by jacobowitz — CHANGES_REQUESTED** [review]
Hi @anuragdw710 
Thank you for our PR!
As @bwanglzu said: Please make sure to give the PR a proper title and description. Please also link any issue if there exists one. Please also read our guide on how to [contribute to Jina](https://github.com/jina-ai/jina/blob/master/CONTRIBUTING.md)

And also please make sure that this change is covered by tests :)

**Comment by jacobowitz** [comment]
The PR looks stale to me, will close it for now. @anuragdw710 please feel free to reopen if you want to address the feedback.

**Pull request closed without merging**